DQM/HLTEvF/python/HLTObjectsMonitor_B2G_cfi.py

```diff
@@ -0,0 +1,83 @@
+import FWCore.ParameterSet.Config as cms
+
+b2gObjects = cms.VPSet(
+       cms.PSet(
```

Comment:
its pretty bad, no? the 40 line psets are repeated many times with little change